tock/hours/models.py

```diff
@@ -330,3 +333,26 @@ def save(self, *args, **kwargs):
 
 
         super(TimecardObject, self).save(*args, **kwargs)
+
+
+class TimecardPrefillData(models.Model):
+    employee = models.ForeignKey(UserData, related_name='timecard_prefill_data')
+    project = models.ForeignKey(Project, related_name='timecard_prefill_data')
+    hours = models.DecimalField(
+        decimal_places=2,
+        max_digits=5,
+        help_text='The amount of hours this user is assigned to this project.'
+    )
+    notes = models.TextField(
+        blank=True,
+        default='',
```

Comment:
Super minor, but is minor required here?

Reply:
I'm not sure what you're referring to, @tbaxter-18f, do you mind expanding your question a bit more?  Are you asking if this field is required?

Reply:
Nothing that large. I was just saying I don't think you need to have `default=''`. But I don't think it's hurting anything, either.